src/common/client-app.ts

```diff
@@ -65,19 +73,12 @@ class App {
 
   onInterval= () => {
     var current = this.boardVM.toPlain();
-    var myChanges = rfc6902.createPatch(this.shadowServer, current);
+    var myChanges = rfc6902.createPatch(this.shadowClient, current);
     if (myChanges.length) {
-      // TODO: consider to send it using HTTP in place of Socket
-      this.socket.emit(this.config.socketEventName, { patch: myChanges });
+      this.sendToServer(myChanges);
     }
   };
 
-  start() {
-    this.boardVM.update(this.shadowServer);
-    this.boardVM.applyBindings(this.config.rootNode);
-    this.shadowClient = this.boardVM.toPlain();
-    this.socket = io();
-    this.socket.on(this.config.socketEventName, this.onMessage);
-    setInterval(this.onInterval, this.config.throttlingInterval);
-  }
+  sendToServer: (changes: any) => void;

```

Comment:
Could you move this declaration upper, [near to the other _class properties_](https://github.com/makingsensetraining/board-poc/pull/12/commits/e4339b7befa094de0be1e7e2591e43fcd77ace09#diff-0da6bc1f089884e71dcd059fadc0924bL33)?